build.gradle

```
@@ -200,6 +202,9 @@ distributions {
             from("build/jre/linux-x64") {
                 into 'lib/jre'
             }
+            from("build/tobiiDriversForLinux/linux-x64") {
+                into 'tobiiDriversForLinux'
+            }
```

@Tohaker I guess the issue is around here...

---

You did everything right up until the spelling of the folder
```suggestion
            from("build/tobiiDriversForLinuxFolder/linux-x64") {
                into 'tobiiDriversForLinux'
            }
```